piptools/scripts/options.py

```diff
@@ -364,3 +371,28 @@ def _get_default_option(option_name: str) -> Any:
     is_flag=True,
     help="Restrict attention to user directory",
 )
+
+build_deps_for = click.option(
+    "--build-deps-for",
+    "build_deps_targets",
+    multiple=True,
+    type=click.Choice(ALL_BUILD_TARGETS),
+    help="Name of a build target to install dependencies for. "
+    "Static dependencies declared in 'pyproject.toml::build-system.requires' will be included as "
+    "well; may be used more than once.",
+)
+
+all_build_deps = click.option(
+    "--all-build-deps",
+    is_flag=True,
+    default=False,
+    help="Install dependencies for all build targets. "
+    "Static dependencies declared in pyproject.toml will be included as well.",
+)
+
+only_build_deps = click.option(
+    "--only-build-deps",
+    is_flag=True,
+    default=False,
+    help="Install a package only if it is a build dependency.",
```

Comment:
I think that using "install" here is inaccurate. Should it be "extract" or something else?

Reply:
I too think it sounds strange but I'm trying to be consistent with the terminology used for the `--extra` flag. Do you think it is worth diverging?

Reply:
I think it is, since we may be consistent either with old unrelated flags or with the new feature. Why not go for the consistency within the newly added code, then?

Reply:
Then the sentences would read like below, right? 4 & 5 could be applied in a separate PR if we want to attempt to align those flags as well. Note that the text in parenthesis does not have to be included, it's there mostly to clarify how I interpret it.

1. Name of a build target to extract dependencies for
2. Extract dependencies for all build targets
3. Extract a package only if it is a build dependency
4. Name of an extras_require group to extract (from package info)
5. Extract all extras_require groups

The only alternative to _extract_ that I can think of is _include_. Then it would read

1. Name of a build target to include dependencies for (in the output from the resolver)
2. Include dependencies for all build targets
3. Include a package only if it is a build dependency
4. Name of an extras_require group to include (in the input to resolver)
5. Include all extras_require groups

Reply:
I think that "extract" can be augmented with "include". Something like "Name of a build target to extract dependencies for, that always implicitly includes statically declared dependencies". But in general, "extract" seems more accurate.

Reply:
I have updated the help texts.

Reply:
Brilliant! Marking this as resolved!